### PR TITLE
make compatible with FEFLOW8.1

### PR DESCRIPTION
**Pull request by HeiJuli**

### CODE REVIEW

**Comment by HeiJuli** [comment]
> @HeiJuli In `pyproject.toml` please change line 15 to (should fix ci):
> 
> ```toml
> dependencies = ["geopandas", "matplotlib<3.8", "numpy", "pandas<2.2", "shapely"]
> ```

did so, but I think you were talking about line 13.

File: pyproject.toml [hunk 1]
```diff
@@ -12,11 +12,7 @@ authors = [{ name = 'Alexander Renz', email = 'alex.renz@gmx.net' }]
 license = { text = "MIT" }
 readme = "README.md"
 requires-python = '<3.12'
-dependencies = ["geopandas", "matplotlib<3.8", "numpy", "pandas", "shapely"]
-
-[project.optional-dependencies]
```
Comment (bilke) [inline pyproject.toml:17]: @HeiJuli You accidentally deleted some lines …

**Review by bilke — CHANGES_REQUESTED** [review]

File: pyproject.toml (hunk 1 above)
Comment (HeiJuli) [reply pyproject.toml:17]: thanks for viewing the mistake!
it should be fixed

Comment (bilke) [reply pyproject.toml:17]: You have moved the line. The dependencies need to be at line 15!

**Review by bilke — CHANGES_REQUESTED** [review]

**Review by bilke — APPROVED** [review]

**Pull request merged**